WORKSPACE

```diff
@@ -8,6 +8,10 @@ load("//bazel:api_repositories.bzl", "envoy_api_dependencies")
 
 envoy_api_dependencies()
 
+load("@rules_buf//buf:repositories.bzl", "rules_buf_toolchains")
```

Comment:
@phlax do you think if it's better to move this to a new function named `envoy_dependency_imports()`?

Reply:
it definitely doesnt want to be in WORKSPACE

strictly speaking its an api dep - so im wondering if it should move somewhere in there

for now tho - i think just add it to the existing fun in bazel/dependency_imports.bzl as that is where all the other toolchains are registered

Reply:
updated